internal/internal_workflow.go

```diff
@@ -1400,37 +1396,8 @@ func (h *queryHandler) execute(input *commonpb.Payloads) (result *commonpb.Paylo
 		}
 	}()
 
-	fnType := reflect.TypeOf(h.fn)
-	var args []reflect.Value
-
-	decoded, err := decodeArgs(h.dataConverter, fnType, input)
-	if err != nil {
-		return nil, fmt.Errorf("unable to decode the input for queryType: %v, with error: %w", h.queryType, err)
-	}
-	args = append(args, decoded...)
-
-	// invoke the query handler with arguments.
-	fnValue := reflect.ValueOf(h.fn)
-	retValues := fnValue.Call(args)
-
 	// we already verified (in validateHandlerFn()) that the query handler returns 2 values
```

Comment:
no longer need this comment

Reply:
While it still applies for why we are not validating the function return types here, I will remove because `executeFunction` also validates.

Reply:
Fixed